packages/react-components/react-alert/src/components/Alert/Alert.test.tsx

```diff
@@ -47,7 +47,11 @@ describe('Alert', () => {
   });
 
   it('renders a button', () => {
-    render(<Alert action={{ children: 'Undo' }}>Test</Alert>);
+    render(
+      <Alert action={{ children: 'Undo' }} appearance="inverted">
```

Comment:
```suggestion
      <Alert action="Undo" appearance="inverted">
```

nit: Similar to @layershifter's comment below, you can use shorthand form for this slot as other props are not used